src/rust/engine/fs/store/src/lib.rs

```diff
@@ -901,6 +908,7 @@ impl Store {
           file_digests.insert(f.digest());
         }
         directory::Entry::Directory(_) => (),
+        directory::Entry::Symlink(_) => todo!(),
```

Comment:
Do nothing here since the goal of this loop is to find file digests so that those digests can be checked for existence. Symlinks always "exist" without such further checks since the symlink exists if and only if it exists as a `SymlinkNode` in a `Directory` proto.

The goal of this method is ensure that file and directory digests are present in the CAS.